.changeset/purple-phones-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/project-access': minor
+---
+
+Add findFioriArtifact() to allow searching for apps, adaptation projects, extension projects, and libraries
```

Comment:
New API function name is findFioriArtifacts(). Missing a 's' in the cset.

Reply:
@slin-sap, thanks, fixed it.